langgenius/dify
# feat: install app from remote

# Description

I am building DifyShare, a platform to share Dify apps. To allow importing them with a button click from DifyShare, this PR introduces the ability to install apps from an url. This functionality can be used for other scenarios too.

## Type of Change

# How Has This Been Tested?
I tested it locally. To test it manually:
- Configure beta.difyshare.com as allowed host by adding this to the .env for the web app: `APP_IMPORT_FROM_REMOTE_ALLOWED_HOSTS="beta.difyshare.com"`
- Visit the Dify Studio (navigate to /apps)
- Add `?action=installApp&remoteInstallUrl=https://beta.difyshare.com/api/flow/663ffe78002cf37178fa/download` to the URL
- Open the URL
  - A loading spinner should be rendered with a message in the users locale
  - The Chatflow from DifyShare should succeed and you see the workflow editor

# Suggested Checklist:
